cloud/pkg/cloudhub/servers/server.go

```diff
@@ -37,13 +35,13 @@ func StartCloudHub(config *util.Config, messageq *channelq.ChannelMessageQueue)
 	handler.InitHandler(config, messageq)
 
 	svc := server.Server{
-		Type:       api.ProtocolTypeQuic,
+		Type:       protocolType,
 		Addr:       fmt.Sprintf("%s:%d", config.Address, config.QuicPort),
 		TLSConfig:  &tlsConfig,
 		AutoRoute:  true,
 		ConnNotify: handler.CloudhubHandler.OnRegister,
-		ExOpts:     api.QuicServerOption{MaxIncomingStreams: config.MaxIncomingStreams},
+		ExOpts:     exOpts,
 	}
```

Comment:
We can add a switch logic here instead of pass as parameter. @du2016 @kadisi  Thought?

Reply:
if add a switch logic,whether should add config parameter to hubconfig?

Reply:
@fisherxu  yes , adding a switch logic in `StartCloudHub()` function looks good.

Reply:
> whether should add config parameter to hubconfig 

No need add to hubconfig, if add to hubconfig, we still need to judge which protocol used.

Reply:
OK ,i'll add a switch logic

Reply:
ok, thanks :) then let's got merged.